demisto_sdk/commands/common/tests/playbook_test.py

```diff
@@ -71,9 +71,9 @@ class TestPlaybookValidator:
                             }
     IS_CONDITIONAL_INPUTS = [
         (CONDITION_NOT_EXIST_1, True),
-        (CONDITION_EXIST_EMPTY_1, False),
-        (CONDITION_EXIST_EMPTY_2, False),
-        (CONDITION_EXIST_PARTIAL_1, False),
+        (CONDITION_EXIST_EMPTY_1, True),
```

Comment:
what did we do here?

Reply:
These conditions are "valid" because it's possible to create a conditional task without conditions / empty conditions (ask, script and manual conditional tasks).